python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -462,9 +462,10 @@ def add_file_paths(
         Each path is repeated according to the number of rows read from it.
         """
         (filepaths,) = plc.filling.repeat(
-            # TODO: Remove call from_arrow when we support python list to Column
+            # TODO: Remove call from_arrow when we support python list[str] to strings Column
+            # Needs https://github.com/rapidsai/cudf/issues/17192
             plc.Table([plc.interop.from_arrow(pa.array(map(str, paths)))]),
-            plc.interop.from_arrow(pa.array(rows_per_path, type=pa.int32())),
+            plc.Column.from_list(rows_per_path, dtype=plc.DataType(plc.TypeId.INT32)),
```

Comment:
Just FYI, I'm just using `pl.Series` for this in https://github.com/rapidsai/cudf/pull/18564/files#diff-58a9bfc6f3a54d841c1a798c95354d72b05555be355adec331ca5d5d1c67ad16R465